react/features/base/conference/middleware.any.ts

```diff
@@ -204,17 +203,7 @@ function _conferenceFailed({ dispatch, getState }: IStore, next: Function, actio
             dispatch(overwriteConfig(newConfig)) // @ts-ignore
                 .then(() => dispatch(conferenceWillLeave(conference)))
                 .then(() => conference.leave())
-                .then(() => dispatch(disconnect()))
-                .then(() => dispatch(connect()))
-                .then(() => {
-                    // FIXME: Workaround for the web version. To be removed once we get rid of conference.js
-                    if (typeof APP !== 'undefined') {
-                        const localTracks = getLocalTracks(getState()['features/base/tracks']);
-                        const jitsiTracks = localTracks.map((t: any) => t.jitsiTrack);
-
-                        APP.conference.startConference(jitsiTracks).catch(logger.error);
-                    }
-                });
+                .then(() => dispatch(disconnect()));
```

Comment:
When will the connection begin again?

Reply:
I see some of the logic got moved to base/conference, but only to the web middleware. When does mobile start connecting again?